docs/guide/08-tools.md

```diff
@@ -125,79 +125,74 @@ Build tools take your code and make it production-ready. Among the things you ma
 
-To build your project with the Polymer CLI, first install the Polymer CLI:
-
-`npm i -g polymer-cli`
-
-Create a `polymer.json` file in your project folder. A simple example would look like this:
-
-```json
-{
-  "entrypoint": "index.html",
-  "shell": "src/myapp.js",
-  "sources": [
-    "src/**.js",
-    "manifest/**",
-    "package.json"
-  ],
-  "extraDependencies": [
-    "node_modules/@webcomponents/webcomponentsjs/bundles/**"
-  ],
-  "builds": [
-    {"preset": "es6-bundled"}
-  ]
-}
-```
-
-This configuration specifies that the app has an HTML entrypoint called `index.html`, has a main JavaScript file (app shell) called `src/myapp.js`. It will produce a single build, bundled but not transpiled to ES5. For details on the polymer.json file, see [polymer.json specification](https://polymer-library.polymer-project.org/3.0/docs/tools/polymer-json) on the Polymer library site.
-
-To build the project, run the following command in your project folder:
-
-`polymer build`
-
-For more on building with Polymer CLI, see [Build for production](https://polymer-library.polymer-project.org/3.0/docs/apps/build-for-production) in the Polymer library docs.
+open-wc also has [Rollup build resources](https://open-wc.org/building/building-rollup.html). 
 
 ### Build your project with webpack
 
+webpack is a powerful build tool with a large ecosystem of plugins. 
 
-See the Open Web Components default webpack configuration provides a great starting point for building projects that use lit-html. See their [webpack page](https://open-wc.org/building/building-webpack.html#default-configuration) for instructions on getting started. 
-
-### Build considerations for other tools {#build-considerations}
+See the open-wc default webpack configuration provides a great starting point for building projects that use lit-html. See their [webpack page](https://github.com/open-wc/open-wc/tree/master/packages/building-webpack) for instructions on getting started. 
 
+### Build considerations  {#build-considerations}
 
 If you're creating your own configuration for webpack, Rollup, or another tool, here are some factors to consider:
 
-* ES6 to ES5 transpilation.
+* ES6 to ES5 compilation.
 * Transforming JavaScript modules to other formats for legacy browsers.
 * lit-html template minification.
+* Polyfills.
+
+For more details on these considerations, see the LitElement [Build for production](https://lit-element.polymer-project.org/guide/build) guide. lit-html has the same requirements as LitElement, except that lit-html requires only the [Template polyfill](#template-polyfill), not the full Web Components polyfills.
 
-#### Transpilation and module transform
+#### Compilation and module transform {#transpilation-and-module-transform}
 
-You build tools need to transpile ES6 features to ES5 for legacy browsers. 
+You build tools need to compile ES6 features to ES5 for legacy browsers. 
```

Comment:
You -> Your, but also, I'm having a little difficulty parsing the intent of this sentence. Is it saying that my tools definitely need to compile my code, or that they may need to if I'm targeting legacy browsers?

Reply:
Mmm. Yep, that should be flipped around. "To support legacy browsers, your build tools need to compile ..."